### PR TITLE
Added locally ran storybook script and readme

### PR DESCRIPTION
**Pull request by vladbalanescu**
Added locally served storybook config and updated the Readme.md accordingly

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +47,14 @@
     "react": "^16.13.1",
     "rimraf": "^3.0.2",
     "rollup": "^2.28.2",
+    "@storybook/addon-actions": "^6.0.6",
```
Comment (lee-chase) [inline package.json:50]: Are these storybook packages needed here?

Comment (asfordmatt) [reply package.json:50]: I dont think we're using all of those

Comment (vladbalanescu) [reply package.json:50]: yep, indeed, will get them out in my next push, cheers guys 👍

Comment (lee-chase) [reply package.json]: Can they go in the core package?

**Review by lee-chase — APPROVED** [review]

**Comment by vladbalanescu** [comment]
@lee-chase @asfordmatt can we get this one merged in now please? 😄

**Review by asfordmatt — APPROVED** [review]
LGTM

**Pull request merged**